### ISSUE TITLE
After Effects Version Error

### ISSUE THREAD

**Issue by mengozq** [inlife/nexrender#968]
Hello Vlad!

I'm getting this error when rendering with AE 2023. Hope you can help me with it.

"rendering reached an error: After Effects error: The file you are attempting to open was created with After Effects version 23.6.2 (Windows 64) and cannot be opened with this version."

AE version 23.6.2
NexRender version 1.47.1

I have included the outputModule and outputExt in my Json.
  "template": {
    "src": "file:///hiddenfilepath/TemplateA.aep",
    "composition": "CompName",
    "settingsTemplate" : "Best Settings",
    "outputModule" : "H.264 - Match Render Settings - 15 Mbps",
    "outputExt" : "mp4"
  },

**Comment by inlife** [comment]
There could be differences between the AE version where the project was created and the AE where the project is being opened. Have you tried opening that AE project file manually? Does it ask you to convert the project? Does it show the same error?

**Comment by mengozq** [comment]
Thanks for the reply.

There's no pop up when I'm opening the file from AE 2023 directly. And I'm
using the same AE version, AE 2023, to create the file. Currently I can
only render with Nexrender after down saving it the a lower version.

**Comment by inlife** [comment]
Could it be there are more than one AE versions installed on your machine, and it uses a different one with nexrender?

**Comment by mengozq** [comment]
That certainly seems to be the case since I first installed nexrexder CLI
when I'm still using AE 2022. If this is so, how do i update nexrender to
use the latest version or if there are any ways for me to state which
version of AE to be used?

**Comment by inlife** [comment]
Please refer to the --binary flag in the --help menu of the cli or worker interface. You will need simply to provide a valid path to the binary and that's it!

**Comment by mengozq** [comment]
Hey Vlad,

I tried adding --binary, but I ran into a couple of problems. With a few questions.
1. I'm using NodeJS, so I'm not executing NexRender via CMD. I tried adding --binary via this.
`        const result = await render(nexRenderMenuData,{
            binary: 'C:\\Program Files\\Adobe\\Adobe After Effects 2023\\Support Files\\aerender.exe'
        });`
I thought it worked since it was asking me to run application with admin priviledges once.

2. How do I run admin using js/nodejs?

3. I didn't know how so I ran cmd in admin and tried to do the same task,
`nexrender-cli-win64.exe --binary="C:/Program Files/Adobe/After Effects CC 2023/Support Files/aerender.exe" --file C:\hiddenpath\NexRenderMenu.json`.
However, it still states "using automatically determined directory of After Effects installation: which is AE 2020.
